### ISSUE TITLE
SpaceExploration + AAI Programmable Vehicle break Vehicle Depot selection priotrity

### ISSUE THREAD

**Issue by KeinNiemand** [KeinNiemand/Factorio-GhostOnWater#39]
The changes in collision masks when SE is also installed break the Selection priority for the Vehicle Depot from AAI, to fix this a patch has to be added that changes the selection priority of the different vehicle depot entitles when both mods are installed to ensure the right entity (the container and maybe the signal) can be selected.
Can't think of a generic way to fix this so I guess a fix would have to be added for every multi entity that breaks due to collision mask/selection priority problems.

**Comment by angrypidgeon123** [comment]
True, just run into this problem. Can't open the depot inventory nor deconstruct it with ghosts on water mod installed

**Comment by KeinNiemand** [comment]
Since this mod is going to be obsolete I'm not going update it anymore unless something affects me personally or someone makes a pull request. I think this is better to be fixed on the AAI side, all AAI would have to do is change the selection priority of their entity to ensure that the inventory of the depot is always on top. It's better if the mod itself set's the selection priority of it's own entities instead of relying on the collision mask as a tie breaker (which causes it to break with ghost on water becosue ghost on water changes collision on everything to ensure you can't place the water ghosts on top of existing buildings.
